### ISSUE TITLE
Received chat packet with missing or invalid signature

### ISSUE THREAD

**Issue by JorelAli** [JorelAli/CommandAPI#311]
### CommandAPI version

8.5.0

### Minecraft version

1.19

### Are you shading the CommandAPI?

No

### What I did

- Register this command:
  ```java
  new CommandAPICommand("test")
        .withArguments(new AdventureChatArgument("testarg")
            .withPreview(previewInfo -> MiniMessage.miniMessage().deserialize(previewInfo.input())))
        .executes((commandSender, objects) -> {
          commandSender.sendMessage("test");
        })
        .register();
  ```
- Run `/test <gray>`

### What actually happened

The player is instantly kicked from the server with the message `Received chat packet with missing or invalid signature`

### What should have happened

The player receives the message "test"

### Other

This occurs only on 1.19.1, but not 1.19.

**Comment by JorelAli** [comment]
Currently being worked on as of f558112ba98cf8646650949fe2a249371021a425 for 8.5.1's release.

**Comment by JorelAli** [comment]
Fixed in 8.5.1.
